workspaces/diff-engine/cli/src/main.rs

```diff
@@ -6,19 +6,18 @@ use num_cpus;
 use optic_diff_engine::diff_interaction;
 use optic_diff_engine::errors;
 use optic_diff_engine::streams;
-use optic_diff_engine::Aggregate;
 use optic_diff_engine::HttpInteraction;
 use optic_diff_engine::InteractionDiffResult;
 use optic_diff_engine::SpecProjection;
-use optic_diff_engine::{RfcCommand, SpecCommand};
-use optic_diff_engine::{RfcEvent, SpecChunkEvent, SpecEvent};
+use optic_diff_engine::{SpecChunkEvent, SpecEvent};
 use std::cmp;
-use std::path::Path;
 use std::process;
 use std::sync::Arc;
 use tokio::io::{stdin, stdout};
 use tokio::sync::mpsc;
 
+mod commit;
+
 fn main() {
```

Comment:
Moved the `commit` subcommand logic to it's own module, planning to do the same for `diff` as well. Logic for both is non-trivial and requires some command specific structs, so using `clap`'s ability to modularise seems like a good thing to get into at this point.